### ISSUE TITLE
Add config entry to disable for repo

### ISSUE THREAD

**Issue by davwheat** [dabutvin/Imgbot#892]
I use Upptime (open source status page) which continuously updates graph images to show response time for my services.

imgbot is constantly attempting to optimise these whenever they're updated, wasting your computational time.

I'd love a simple `enabled` key in the JSON config to entirely disable imgbot for a single repo.

**Comment by selul** [comment]
@davwheat when you install the app, you have the option to enable on which repositories you want imgbot to run, removing the ones that you don't need.

Will it work for your usecase?

**Comment by davwheat** [comment]
Yes, I suppose. I do enjoy having it on by-default for new repos when I create them. Changing to "Select repositories" would mean I'd have to manually enable it when I have new repositories.
